src/App.js

```diff
@@ -24,32 +24,58 @@ const App = () => {
   const drawCard = () => {
     // 1)
     const draw = `${rank[Math.floor(Math.random() * rank.length)]} ${
-      suit[Math.floor(Math.random() * suit.length)]
+      suit[Math.floor(Math.random() * suit.length)] 
     }`
+    // In number 1, the const will produce a random card by suit and face value. Math.floor, Math.random and arry.length make a diffrent index for rank and suit arrays casuing a diffret card to be chossen. 
+
     // 2)
     if (hand.indexOf(draw) === -1) {
+
+    // the indexOf checks to see if the drawn card is in your hand or not. the === -1, looks for a true or false value. if its -1 the card is not in the hand
+
       // 3)
       setHand([...hand, draw])
+
+    // this shows that the setHand adds a card to the players hand if it is diffrent. the spread operator (...) will crete a new array adding a new card in the current card hand.
+
       // 4)
     } else if (hand.length !== 52) {
+
+    // this will show that the hand is up to date
+
       // 5)
       drawCard()
+
+    // as long as the cards are not all drawn or a copy of the card is not in the hand, it continues to draw untill stoped by a duplicate. 
+
       // 6)
     } else {
+
+    // the else alows a it to draw a unique card, breaking down a large problom into a small one. 
+
       // 7)
       alert("All cards have been dealt.")
     }
   }
+
+    // the string alearts the player that all cards have been dealt once it reaches the length of the deck ().
+
   // No need to change anything ABOVE this line ^
 
-  const shuffle = () => {}
+  const shuffle = () => {
+   setHand([]);
+  };
 
   return (
-    <h1>Draw a Card</h1>
-    <button onClick={drawCard}>Click to Draw a Card</button>
-    <button>Return Cards and Shuffle Deck</button>
+    <>
+    <div>
+      <h1>Draw a Card</h1>
+      <button onClick={drawCard}>Click to Draw a Card</button>
+      <button onClick={shuffle}>Return Cards and Shuffle Deck</button>
     <Card hand={hand} />
-  )
+  </div>
+    </>
```

Comment:
Looks like you doubled up here on the enclosing tags. You can use either the fragments or the div but both are a bit redundant.